pkg/ibmvpc/analysis_output_test.go

```diff
@@ -678,6 +678,20 @@ var tests = []*vpcGeneralTest{
 		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
 		format:      vpcmodel.Text,
 	},
+	{
+		inputConfig:  "filters_split_lb_subnet",
+		useCases:     []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
+		format:       vpcmodel.Text,
+		grouping:     false,
+		noLbAbstract: true,
```

Comment:
can you add this test also with `noLbAbstract: false` ? 
currently the only cli output available is the one with abstraction, right? 
also, can you add in comment some details on this example- what are the filters that cause the split of the subnets?  does it cover both sg and acls?

Reply:
also, can you add details on which connections in the concrete version are different from the abstraction version?

Reply:
done